docs/architecture/interconnect.rst

```diff
@@ -0,0 +1,26 @@
+Interconnect PIPs
+=================
+
+Fake PIPs
+---------
+
+Some PIPs are not "real", in the sense that no bit pattern in the bit-stream correspond to the PIP being used. This is the case for all the PIPs in the switchbox in a CLB tile (ex: CLBLM_L_INTER): They either correspond to buffers that are always on (i.e. 1:1 connections such as `CLBLL_L.CLBLL_L_AQ->CLBLL_LOGIC_OUTS0`), or they correspond to permutations of LUT input signals, which is handled by changing the LUT init value accordingly, or they are used to "connect" two signals that are driven by the same signal from within the CLB.
```

Comment:
> they correspond to permutations of LUT input signals, which is handled by changing the LUT init value accordingly, or they are used to "connect" two signals that are driven by the same signal from within the CLB.

I think this sentence is actually false.  Do you have the evidence that supports this conclusion?

Reply:
I added a FIXME to look into this. I believe what is mentioned here is actually true.